andela/coding-geeks-ah-frontend
# #169646191: Fix jwt malformed error

### What does this PR do?
- Fixes a bug of jwt malformed error

### How should this be manually tested?
- Clone this branch
- `npm install`
- Try change the token with an invalid token, when you do an action you should be redirected to login page
### What are the relevant pivotal tracker stories?
- [ Fix jwt malformed error](https://www.pivotaltracker.com/story/show/169646191)
